packages/loctool/loctool.js

```diff
@@ -430,9 +430,13 @@ for (var i = 0; i < argv.length; i++) {
             usage();
         }
     } else if (val === "--xliffStyle") {
-        var candidate = ["standard", "custom"];
-        if (candidate.indexOf(argv[i+1]) !== -1) {
+        var candidate = XliffFactory.getAllStyles();
```

Comment:
commandHelpOptions above is not updated with this change.

Reply:
I updated it